contracts/math/SafeMath.sol

```diff
@@ -15,18 +19,27 @@ library SafeMath {
     return c;
   }
 
+  /**
+  * @dev Integer division of two numbers, truncating the quotient.
+  */
   function div(uint256 a, uint256 b) internal pure returns (uint256) {
     // assert(b > 0); // Solidity automatically throws when dividing by 0
     uint256 c = a / b;
     // assert(a == b * c + a % b); // There is no case in which this doesn't hold
     return c;
   }
 
+  /**
+  * @dev Substracts two numbers, throws if subtrahend is greater than minuend.
```

Comment:
I love the specificity but can we change this for the following?
> throws on overflow (i.e. if subtrahend is greater than minuend).

Reply:
Fixed!